microsoft-edge/web-platform/ad-selection-api.md

```diff
@@ -0,0 +1,297 @@
+| `reportResult()` | **SellerFrontEnd** | For event-level reporting.  Runs in the **SellerFrontEnd** service after the final winner has been chosen.  Notifies the seller about the winning bidder, and provides the bid value. |
+
+
+<!-- ---------- -->
+###### User-Defined Functions for buyers
+
+| UDF | Service in which the UDF runs | Description |
+| --- | --- | --- |
+| `generateBids()` | **Bidding** | Generates a bid, and chooses the banner that will be shown. |
+| `reportWin()` | **SellerFrontEnd** | For event-level reporting.  Runs in the **SellerFrontEnd** service, if the buyer's bid wins the auction.  Notifies the buyer that they have won the auction, and generates notification URLs that will be triggered later, such as during banner rendering or other client events. |
+| `getValues()` | **Bidding Selection & Key/Value** | Runs in the **Bidding Selection & Key/Value** service, if the buyer uses that service.  Allows for custom code execution for data lookup and selection. |
+
+
+<!-- ------------------------------ -->
+#### Deployment guide
+
+To deploy the Ad Selection API service to Azure:
+
+1. Create an Azure account and create a subscription at [Microsoft Azure](https://azure.microsoft.com).
+
+1. Choose the **Confidential ACO containers** option.
+
+1. Download the Terraform [deployment scripts and guide](https://go.microsoft.com/fwlink/?linkid=2290115).  **Note:** Make sure the hash of the image you downloaded matches the hash in the script you download.
+
+1. The Terraform scripts already include links to public images that are provided by Microsoft.  Choose the required services and deploy them by following the steps in the included `readme.md`.
+
+   1. Set any specific flags for your environment, including auction worklet locations.
+
+   1. Configure all prerequisites and prepare all tools for deployment.
+
+   1. Create Azure Resources and run the Terraform script that finishes deployment.
+
+1. Develop worklets including `user-defined-function` (UDF) and start Ad Selection API services.
+
+
+<!-- ====================================================================== -->
+## Use the Ad Selection API on your website
+
+To start using the Ad Selection API, sellers and buyers need to modify the JavaScript code of their websites running in Microsoft Edge.
+
+* Sellers: To run private actions, fetch interest groups, and do reporting, update your supply-side and publisher code.
+
+* Buyers: To create interest groups and store them in the browser, update the code on your advertiser partners' websites.  You can use the delegation mechanism to restrict and allow Interest Group creation on third-party domains.
+
+To learn more about which API methods are available and to review example code, see [Ad Selection API details](https://github.com/WICG/privacy-preserving-ads/blob/main/API%20Details.md).
```

Comment:
```suggestion
To learn more about which API methods are available and view example code, see [Ad Selection API details](https://github.com/WICG/privacy-preserving-ads/blob/main/API%20Details.md).
```
We mean "see, inspect, for the first time; study".  The word "review" means read something that you have already read before.
I picked 'view', because 'see' is already present.

https://ahdictionary.com/word/search.html?q=review --
![image](https://github.com/user-attachments/assets/23fd3705-1ef4-4190-9d52-40355a8e3b75)
@captainbrosset